public/index.html

```diff
@@ -10,11 +10,41 @@
 </head>
 <body>
     <ul class="timeline">
-        <li class="event">Time of event
-            <ul>
-                <li>Name of event</li>
-                <li>Description of event</li>
-            </ul>
+        <li class="event">
+            <div>
```

Comment:
Make this more explicit by adding a class that explains what this `div` does and what content is inside of it.